### ISSUE TITLE
Analyzing effect of dimensionality reduction on accuracy of different classifiers on different types of datasets

### ISSUE THREAD

**Issue by parimal173** [neurodata/scikit-learn#4]
On High dimensional dataset of different types, does dimensionality reduction improves the performance of Classification in case of different classifiers:
Questions to answer:
1. Comparing the performance of different dimensionality reduction techniques between different types of datasets
2. Analyzing how dimensionality reduction helps in classification for different classifiers.

Pipeline to be followed: 
1. Defining a dataset with sklearn established synthetic datasets. 
2. Performing classification on data and measuring accuracy for quantification of the process.
3.performing Dimensionality reduction technique.
4. Checking the performance of classification again after dimensionality reduction. 

Experiments to follow:
https://github.com/NeuroDataDesign/team-forbidden-forest/tree/master/Parimal%20Joshi

**Comment by bdpedigo** [comment]
This issue is unclear about what you are actually proposing to PR into sklearn. You can be a lot more detailed about the fact that you are proposing a new tutorial, what the figures will be, what the data is

**Comment by parimal173** [comment]
> This issue is unclear about what you are actually proposing to PR into sklearn. You can be a lot more detailed about the fact that you are proposing a new tutorial, what the figures will be, what the data is:

I have made changes in the issue, please tell me if its enough
